server/formatter.js

```diff
@@ -4,6 +4,8 @@ const qs = require('querystring')
 const unescape = require('unescape')
 const list = require('./list')
 
+/* Your one stop shop for all your document processing needs. */
```

Comment:
I think it's worth reconsidering some of our module names. Maybe formatter.js → docProcessor.js and categories.js → documents.js?

Reply:
For another PR, will make an issue.